modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
+==== Fixing the `table_style_by_css` option
+
+Previously, tables were styled using CSS rather than attributes, aligning with modern web development practices.
+
+However, due to {productname}'s legacy predating this shift, a default option called `table_style_by_css` was introduced in {productname} 6 to accommodate this transition. Unfortunately, the implementation of this feature resulted in an unexpected behavior where the older border attribute was not updated alongside CSS changes, causing issues such as the disappearance of dashed outlines around tables with no borders.
+
+As a consequence, a breaking change has been initiated to address this issue, and the below changes are:
+
+* when a table's border `width` is set to `zero`, the **border attribute** will also be set to `zero`.
+
+=== Removed
+
+[[ui-and-ux-elements-and-components-force-notifications-to-have-a-close-button]]
+==== `Force` notifications to have a close button
+
+Currently, {productname} has the option to modify default notifications so that they do not have display a `X` (close button).
+
+{productname} is committed to enhance accessibility for all users, and aim to create consistency across all notifications and establish a standard tabbing flow for keyboard users.
+
+As of {productname} 7.0, the option `closeButton` has been removed and it will be defaulted to `true`.
+
+[IMPORTANT]
+If customers in-counter table styling issues, they will need to update their CSS to match the **new pattern** produced by {productname}.
 
 [[security]]
 == Security
 
-...
+=== Changed
+
+[[sandbox-iframes-option]]
+==== `sandbox_iframes`
+
+Previous Security penetration testing surfaced a security issue with {productname} loading iframes `unsandboxed` which as a result could allow arbitrary code execution inside {productname} content.
+
+As a result, a configuration option was introduced in {productname} 6.8 to sandbox `(to isolate)` iframes and prevent javascript from executing code inside {productname} documents.
+
+The config option is currently set to `opt in`.
+
+{productname} 7.0, the default value of the `sandbox_iframes` configuration option will be changed from `false` to `true`, making it `opt out` instead of `opt in` for their editors.
+
+[IMPORTANT]
+The premium **Enhanced Media Embed** plugin uses `iframes` extensively and are currently no workarounds available except to keep a record of allowed sources. Customers who want to continue to use **Enhanced Media Embed** plugin in {productname} 7.0 will be required to add the option `sandbox_iframes: false` to their {productname} init configuration.
+
+.Example
+[source, js]
+----
+tinymce.init({
+  selector: "textarea",
+  sandbox_iframes: false,
+});
+----
+
+This follows our policy that we ship the most secure version of {productname} possible, while giving our customers the opportunity to make informed decisions on dialing back protections to a level that fits their use-case.
```

Comment:
@MitchC1999 see below to match release notes.

```suggestion
In {productname} 6.8.1, the xref:content-filtering.adoc#sandbox-iframes[sandbox iframes] editor option was introduced to allow iframes to be sandboxed by default when inserted into the editor.

In {productname} 7.0, the default for `+sandbox_iframes+` will change from `false` to `true`, meaning that all `+iframe+` elements inserted into the editor will be given the `sandbox=""` attribute by default, preventing most actions, including scripting and same-origin access, which may break existing editor content or produce undesirable effects.

To prevent any expected iframes from being sandboxed, we recommend adding the source domains of such iframes to the new xref:content-filtering.adoc#sandbox-iframes-exclusions[`+sandbox_iframes_exclusions+`] option list, and including the domains in the default list where necessary. To prevent all iframes from being sandboxed, set the option `+sandbox_iframes+` to `+false+` in your editor configuration.

For further details on the `+sandbox_iframes+` option, see the xref:content-filtering.adoc#sandbox-iframes[the content filtering options], or refer to the xref:security.adoc#sandbox-iframes[security guide], or the link:https://www.tiny.cloud/docs/tinymce/6/6.8.1-release-notes/#new-sandbox_iframes-option-that-controls-whether-iframe-elements-will-be-added-a-sandbox-attribute-to-mitigate-malicious-intent[{productname} 6.8.1 release notes].
```